lib/cartopy/tests/mpl/test_set_extent.py

```diff
@@ -135,20 +135,20 @@ def test_view_lim_autoscaling():
     y = np.linspace(0.03739792, 0.33029076)
     x, y = np.meshgrid(x, y)
     ax = plt.axes(projection=ccrs.RotatedPole(37.5, 357.5))
-
     plt.scatter(x, y, x * y, transform=ccrs.PlateCarree())
 
     expected = np.array([[86.12433701, 52.51570463],
                          [86.69696603, 52.86372057]])
 
-    assert_array_almost_equal(ax.viewLim.frozen().get_points(), expected)
+    assert_array_almost_equal(ax.viewLim.frozen().get_points(), expected,
+                              decimal=2)
     plt.draw()
-    assert_array_almost_equal(ax.viewLim.frozen().get_points(), expected)
-    ax.relim()
```

Comment:
Matplotlib seems to have changed behaviour in this regard - it seems that relim doesn't handle scatter (a Collection). It may have been fluke that it came up with sensible numbers before (as in, the ```ignore_existing_data_limits``` wasn't being correctly dealt with).